dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops

```diff
@@ -544,4 +544,24 @@ REQUEST_URL_NOT_VALID.useraction=Verify that the request URL that is specified i
 
 INVALID_CONFIG_PARAM=CWWKS5500E: A required configuration parameter {0} is missing or has an invalid value {1}
 INVALID_CONFIG_PARAM.explanation=The social login runtime code did not find a required parameter when processing the client request.
-INVALID_CONFIG_PARAM.useraction=Check the missing parameter in the configuration and make sure that it is specified and correct.
\ No newline at end of file
+INVALID_CONFIG_PARAM.useraction=Check the missing parameter in the configuration and make sure that it is specified and correct.
+
+OIDC_CLIENT_DISCOVERY_SSL_ERROR=CWWKS5501E: The social login feature failed to obtain Open ID Connect Provider endpoint information through the discovery endpoint URL [{0}]. Update the configuration for the oidcLogin with the correct HTTPS discovery endpoint URL. 
+OIDC_CLIENT_DISCOVERY_SSL_ERROR.explanation=The oidcLogin is configured to discover the OpenID Connect Provider endpoints through the discovery URL, but the discovery process failed. The social login feature will be unable to process the authentication requests until the configuration of the discoveryEndpoint is corrected or the discovery is successful.
+OIDC_CLIENT_DISCOVERY_SSL_ERROR.useraction=Correct the oidcLogin configuration to ensure that 1) the discovery endpoint URL refers to the correct OpenID Connect provider, 2) the discovery endpoint URL is HTTPS and 3) the SSL feature and keystore elements are configured correctly with trust for OpenID Connect Provider 4) ensure that the OpenID Connect provider discovery endpoint is functional.
+
+OIDC_CLIENT_DISC_RESPONSE_ERROR=CWWKS5502E: A successful response was not returned from the URL [{0}]. This is the [{1}] response status and the [{2}] error from the discovery request.
```

Comment:
Change "URL [{0}]." to "URL of {0}."

Chage "This is the [{1}] response status and the [{2}] error from the discovery request." to "The {1} response status and the {2} error are from the discovery request.".

Reply:
done